src/uct/ib/base/ib_device.h

```diff
@@ -189,6 +189,18 @@ typedef struct {
 } uct_ib_device_gid_info_t;
 
 
+typedef struct {
+    unsigned           event_type;
+    union {
+        unsigned       port_num;
+        unsigned       qp_num;
+        unsigned       dct_num;
+        struct ibv_cq  *cq;
+        struct ibv_srq *srq;
```

Comment:
we can remove cq/srq pointers and replace by "void * cookie" since on DevX these could not be filled anyway